git-hooks/pre-commit.sh

```diff
@@ -73,7 +73,7 @@ exec 1>&2
 
 $CLANG_FORMAT -version | grep -Fq "$CLANG_FORMAT_VERSION" || (echo Incorrect version of clang-format.; exit 1)
 
-patchfile=$(mktemp --tmpdir --suffix=.patch git-clang-format.XXXXXX)
+patchfile=$(mktemp -t git-clang-format.XXXXXX.patch)
```

Comment:
Thanks @catap , I've been carrying my own patch for macOS to allow the pre-commit hook to run, and happy to get rid of it 👍